Module2_4/Module2_4/Entities/Triangle.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Module2_4.Entities
+{
+    public class Triangle : Shape
+    {
+        public double SideLength { get; set; }
+        
+        public override void InitializeByParameter(double parameter)
+        {
+            SideLength = parameter;
+            Perimeter = SideLength * 3;
+            Area = Math.Pow(SideLength, 2) * Math.Sqrt(3) / 4;
+        }
+
+        public override void InitializeByPerimeter(double perimeter)
```

Comment:
GetPerimeter

Reply:
I guess Get method returns  smth, but it doesn't

Reply:
Ok, You're right.